.travis.yml

```diff
@@ -51,9 +51,6 @@ jobs:
       python: "2.7"
 
     # QA checks.
-    - env: TOXENV=checkqa
```

Comment:
In `checkqa` we had black+isort+flake8 checking. Maybe `isort` should be also moved to fussyfox also and `checkqa` renamed to `black`.

Reply:
Since you seem to be already on it, just do the changes that you think are best.

Reply:
@codingjoe isort checking has been failed. Investigating.